app/javascript/components/UI/Menu.js

```diff
@@ -1,26 +1,231 @@
+      case 'productivity-bad-follow-up':
+        if (activeImg === true) {
+          return {
+            src: complete35_act,
+            percent: 35,
+          };
+        } else {
+          return {
+            src: complete35,
+            percent: 35,
+          };
+        }
+      case 'causes-to-celebrate':
+        if (activeImg === true) {
+          return {
+            src: complete45_act,
+            percent: 45,
+          };
+        } else {
+          return {
+            src: complete45,
+            percent: 45,
+          };
+        }
+      case 'recognition':
+        if (activeImg === true) {
+          return {
+            src: complete65_act,
+            percent: 65,
+          };
+        } else {
+          return {
+            src: complete65,
+            percent: 65,
+          };
+        }
+      case 'icebreaker-answer':
+        if (activeImg === true) {
+          return {
+            src: complete85_act,
+            percent: 85,
+          };
+        } else {
+          return {
+            src: complete85,
+            percent: 85,
+          };
+        }
+      case 'icebreaker-question':
+        if (activeImg === true) {
+          return {
+            src: complete90_act,
+            percent: 90,
+          };
+        } else {
+          return {
+            src: complete90,
+            percent: 90,
+          };
+        }
+        //TODO check after the merge to master branch
+      case 'results':
+        if (activeImg === true) {
+          return {
+            src: complete100_act,
+            percent: 100,
+          };
+        } else {
+          return {
+            src: complete100,
+            percent: 100,
+          };
+        }
+        default:
+          return "";
+        }
+    }
 
-const Menu = ({ percentCompletion, className = ''  }) => {
   return (
     <div className={ `${className}` }>
-      <Dropdown>
+      <Dropdown onClick={handleChangeImg}>
         <Dropdown.Toggle  id='dropdown-stick'>
-        <div  className="menu ">
-          <div  className="m-cover  justify-content-end"></div>
-          <div className='one-line-menu'></div>
-          <div className='one-line-menu'></div>
-          <div className='one-line-menu'></div>
-        </div>
+          <img src={getSrcMenu(lastSegment).src} alt='complete' />
         </Dropdown.Toggle>
         <Dropdown.Menu>
-          <Dropdown.Item href="#" ><Button className='btn-item-menu wb1 mx-auto my-auto' onClick={()=>{}}>Manage Profile</Button></Dropdown.Item>
-          <Dropdown.Item href="#" ><Button className='btn-item-menu wb1  mx-auto my-auto' onClick={()=>{}}>Save Draft</Button></Dropdown.Item>
-          <Dropdown.Item href="#" ><Button className='btn-item-menu wb1  mx-auto my-auto' onClick={()=>{}}>Log Out</Button></Dropdown.Item>
+          {/*<Dropdown.Item href="#" ><Button className='btn-item-menu wb1 mx-auto my-auto' onClick={()=>{}}>Manage Profile</Button></Dropdown.Item>*/}
```

Comment:
Please delete commented out code

Reply:
I wanted to know first whether this button in the project or not. On the first layout in the Figma it is, but in the epic "CI-305: Drop Down" it is not